mirrord-layer/tests/apps/app_node.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const process = require("process");
+const app = express();
+const PORT = 80;
+var done = new Array(4).fill(false);
+
+
+function exit() {
+  server.close();
+  process.exit();
+}
+
+app.get("/", (req, res) => {
+  console.log("GET: Request completed");
+  res.send("GET");
+  done[0] = true;
+  if (done[1] && done[2] && done[3])
+    exit();
+});
+
+app.post("/", (req, res) => {
+  console.log("POST: Request completed");
+  res.send("POST");
+  done[1] = true;
+  if (done[0] && done[2] && done[3])
+    exit();
+});
+
+app.put("/", (req, res) => {
+  console.log("PUT: Request completed");
+  res.send("PUT");
+  done[2] = true;
+  if (done[0] && done[1] && done[3])
```

Comment:
Probably a shorter way to do this, e.g. https://stackoverflow.com/questions/53897673/check-if-all-values-in-array-are-true-then-return-a-true-boolean-statement-jav

Besides being shorter, it would let you add more use cases without having to add to this hard coded list.

Reply:
Changed all apps not to used a hardcoded number of methods.